system/jlib/jlog.cpp

```diff
@@ -3537,19 +3537,18 @@ IRemoteLogAccess * queryRemoteLogAccessor()
         libName.append("lib").append(type.str()).append("logaccess");
 
         //Load the DLL/SO
-        HINSTANCE logAccessPluginLib = LoadSharedObject(libName.str(), true, false);
+        HINSTANCE logAccessPluginLib = LoadSharedObject(libName.str(), false, true);
         if(logAccessPluginLib == nullptr)
             throw makeStringExceptionV(-1, "%s cannot load library '%s'", methodName, libName.str());
```

Comment:
minor: check no longer needed (the last param of LoadSharedObject enforces the check)